iree/hal/cuda/native_executable.c

```diff
@@ -86,6 +92,7 @@ iree_status_t iree_hal_cuda_native_executable_create(
     executable->entry_functions[i].block_size_x = block_sizes_vec[i].x;
     executable->entry_functions[i].block_size_y = block_sizes_vec[i].y;
     executable->entry_functions[i].block_size_z = block_sizes_vec[i].z;
+    executable->executable_layouts[i] = executable_spec->executable_layouts[i];
```

Comment:
you'll want to retain the layout on the executable and have the executable release it when done; see
https://github.com/google/iree/blob/2f7888bc4491e578ae73637613820be012ac8a3b/iree/hal/local/local_executable.c#L23 and https://github.com/google/iree/blob/2f7888bc4491e578ae73637613820be012ac8a3b/iree/hal/local/local_executable.c#L40

Reply:
thanks, added those here and in the destroy function